src/primitives/index.js

```diff
@@ -0,0 +1,36 @@
+import reactPrimitives from 'react-primitives'
+import glamorous, {ThemeProvider, withTheme} from '../index'
+
+// Create glamorous component with composition
+function createComponent(componentConstructor, tag) {
```

Comment:
Sure, we can totally do that but as we are not using any type system in the codebase so I thought it would be better if we could fix the invalid 'tag' type.